webhook-subscription-2022.md

```diff
@@ -1,22 +1,30 @@
-# Solid Webhook Notifications
+# WebHookSubscription2022
 
-Status: Proposal
+## Abstract
+The [Solid Notification Protocol](https://solid.github.io/notifications/protocol) defines a set of interaction patterns for agents to establish subscriptions to resources in a Solid Storage.
 
-## Introduction
+This specification defines a subscription type that applies these patterns to WebHooks.
 
-The Solid Notifications Specification allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
+## Status of This Document
+TODO
+
+## 1. Introduction
+_This section is non-normative._
+
+The [Solid Notifications Protocol](https://solid.github.io/notifications/protocol) allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
 
 Webhooks are an alternative to websockets that are more useful for server-based use cases. While a websocket solution would require a server to maintain a constant websocket connection with a Pod as long as it wants notifications, webhooks only require one registration.
 
-This document proposes an implementation for the `WebHookSubscription2021` Solid Notification protocol.
+This document proposes an implementation for the `WebHookSubscription2022` Solid Notification protocol.
```

Comment:
Is there a reason to change this type? It's fine to leave it as-is, if you'd prefer.

Reply:
I figured it's 2022, and this is a different version. There are servers floating around out there with that follow the old spec I wrote, so changing the version number would help us differentiate between them.

Reply:
We agreed in the 2022-03-24 meeting to use `2021` instead of `2022` ( https://github.com/solid/notifications-panel/pull/52 ). I'd add that:

An Editor's Draft (without a "published" version floating around) is still at an early stage in the process. I think it'd do those servers a favour if they didn't need to carry the baggage around with 2021 if 2022 comes out. 2021 will have to be removed or renamed/updated into 2022 in any case. Sticking to 2021 will just mean that they'll update their current code and get on with the current state of the spec.

Reply:
This comment is outdated, [562e888](https://github.com/solid/notifications/pull/51/commits/562e888a33681309c087acfb164702bbc3993a85) already reuses 2021